queries/cpp/locals.scm

```diff
@@ -72,5 +72,6 @@
 
 (catch_clause) @scope
 
-(destructor_name
-  name: (_) @constructor)
+;; TODO
+;; (destructor_name
+;;   name: (_) @constructor)
```

Comment:
We do have a `@constructor` group

https://github.com/nvim-treesitter/nvim-treesitter/blob/9c456edb3ad24be92a54b42545df8f0d0ee4c71b/lua/nvim-treesitter/highlight.lua#L42-L42

Reply:
Then I need to add that in the `CONTRIBUTING.md`